fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -720,6 +681,31 @@ public Node getNodeVersion(final String label) {
 
     }
 
+    /**
+     * Helps ensure that there are no terminating slashes in the predicate.
+     * A terminating slash means ModeShape has trouble extracting the localName, e.g., for
```

Comment:
Note: @osmandin has a very nice writeup of Modeshape's behavior in the comments for this ticket: https://jira.duraspace.org/browse/FCREPO-1409